flashinfer/topk_varlen/kernels/gvr_topk_decode.py

```
@@ -1322,14 +1575,33 @@ def phase3_collect_candidates(
                     smem_input=smem_input,
                     do_cluster_sync=do_cluster_sync,
                 )
+            cute.arch.barrier()
+
+            # Collapsed tie plateau (> kCC at an adjacent bracket): hand the
+            # row to the existing done=3 machinery; Phase 4 fills from the
+            # tie class in [val_lo, val_hi).
+            mid_chk, adj_chk = order_key_mid_f32(s_thr[1], s_thr[2])
```

Ruff (0.16.2)

[warning] Unpacked variable `mid_chk` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)